crates/provider-wit-bindgen/src/lib.rs

```diff
@@ -1898,6 +2389,58 @@ fn process_fn_arg(arg: &FnArg) -> anyhow::Result<(Ident, TokenStream)> {
     Ok((arg_name, type_name))
 }
 
+/// A trait that represents things that can be converted to a Rust type
+trait ToRustType {
+    /// Convert to a Rust type
+    fn to_rust_type(&self, cfg: &ProviderBindgenConfig) -> anyhow::Result<TokenStream>;
+}
+
+impl ToRustType for wit_parser::Results {
+    fn to_rust_type(&self, cfg: &ProviderBindgenConfig) -> anyhow::Result<TokenStream> {
+        match self {
+            // Convert a named return (usually simple, either empty or tuple)
+            wit_parser::Results::Named(params) => {
+                match params[..] {
+                    // No results
+                    // i.e. `func(arg: string)`
+                    [] => Ok(quote::quote!(())),
+                    // Multiple results i.e. a named type tuple
+                    // i.e. `func(arg: string) -> ( string, string )`
+                    ref params => {
```

Comment:
It looks like this case also handles a single named type, and encloses it in parentheses. Should the comment be updated?

Reply:
Yeah this needs to be updated, will add another example